tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -6606,7 +6606,7 @@ bool HLSLExternalSource::MatchArguments(
   argTypes.clear();
   const bool isVariadic = IsVariadicIntrinsicFunction(pIntrinsic);
 
-  static const UINT UnusedSize = 0xFF;
+  static const UINT UnusedSize = UINT_MAX;
```

Comment:
Is this a fix for #7297?

Reply:
I believe so. I have a small update to this line coming to not use the Windows typedefs and use C++ limits.